### ISSUE TITLE
Redundant input key/value pairs

### ISSUE THREAD

**Issue by arnikz** [LINNAE-project/SFB-Annotator#56]
@lisestork: The input fields seem redundant (differ only in case). Do you agree?

For example,

https://github.com/LINNAE-project/SFB-Annotator/blob/a79d9a5281fbe011fabc2121d115cc82193ff8c8/data/json/local/example_1_1.json#L3

https://github.com/LINNAE-project/SFB-Annotator/blob/a79d9a5281fbe011fabc2121d115cc82193ff8c8/data/json/local/example_1_1.json#L23

Query all `example*.json` files.

```
$ jq -r '[.text,.type]|@csv' data/json/local/*.json
"Taxon","taxon"
"Taxon","taxon"
"Taxon","taxon"
"person","person"
"Person","person"
"Person","person"
"Person","person"
"location","location"
"Location","location"
"measurementorfact","measurementorfact"
"measurementorfact","measurementorfact"
"propertyorattribute","propertyorattribute"
"propertyOrAttribute","propertyorattribute"
"anatomicalentity","anatomicalentity"
"anatomicalEntity","anatomicalentity"
"date","date"
"date","date"
```
Moreover, the RDFizer uses only the `type` field.

https://github.com/LINNAE-project/SFB-Annotator/blob/a79d9a5281fbe011fabc2121d115cc82193ff8c8/src/main/java/localdomain/localhost/servlets/writeAnnotationsToRDF.java#L71

**Comment by arnikz** [comment]
@lisestork: The input fields seems redundant (upper/lowercase). Do you agree?

**Comment by lisestork** [comment]
Yes indeed, I am not sure why the "text" is there, but "type" is the preferred field. Is it maybe from the field that is filled in automatically upon selection of a type? In any case, this information does not need to be present in the json file
